test/test_expect.jl

```diff
@@ -12,6 +12,7 @@ using ITensorNetworks:
 using SplitApplyCombine: group
 using StableRNGs: StableRNG
 using Test: @test, @testset
+using TensorOperations
```

Comment:
```suggestion
using TensorOperations: TensorOperations
```
To minimize what we bring into the namespace.